bin/gobblin-mapreduce.sh

```diff
@@ -4,6 +4,9 @@
 ############### Run Gobblin Jobs on Hadoop MR ################
 ##############################################################
 
+# Set during the distribution build
+GOBBLIN_VERSION=@project.version@
```

Comment:
@sahilTakiar  that will be dangerous as we can't control which version will get loaded. Did you apply the build.gradle changes below ? That seems to be a standard way to replace tokens in a file: https://docs.gradle.org/current/dsl/org.gradle.api.tasks.Copy.html#org.gradle.api.tasks.Copy:filter(groovy.lang.Closure)